java/client/src/main/java/glide/api/commands/GenericClusterCommands.java

```diff
@@ -151,6 +153,85 @@ public interface GenericClusterCommands {
      */
     CompletableFuture<String> randomKey();
 
+    /**
+     * Incrementally iterates over the keys in the Cluster.
+     *
+     * <p>This command is similar to the SCAN command, but it is designed to work in a Cluster
+     * environment. For each iteration the new cursor object should be used to continue the scan.
+     * Using the same cursor object for multiple iterations will result in the same keys or unexpected
+     * behavior. For more information about the Cluster Scan implementation, see <a
+     * href="https://github.com/aws/glide-for-redis/wiki/General-Concepts#cluster-scan">Cluster
+     * Scan</a>. As with the SCAN command, the method can be used to iterate over the keys in the
+     * database, to return all keys the database have from the time the scan started till the scan
+     * ends. The same key can be returned in multiple scans iteration.
```

Comment:
```suggestion
     * finishes. The same key can be returned in multiple scans iteration.
```

Reply:
We should mention that `releaseCursorHandle()` should be called on any cursor that it is no longer needed. This will render the cursor unusable and free memory.